Dangerfile

```diff
@@ -1,5 +1,15 @@
 # frozen_string_literal: true
 
+# Skip Danger run if the opened pull request has "wip" label, [WIP] in the title, or status is Draft
+has_wip_label = github.pr_labels.any? { |label| label.downcase.include? 'wip' }
```

Comment:
@mosharaf13 , I was wondering whether we should move these conditions to the GH-action: there is already an `if` condition existing there:

https://github.com/nimblehq/rails-templates/blob/46573c27d4cbce5dcb9c1f8d3ff2984256f44799/.template/addons/github/.github/workflows/test.yml.tt#L147

I  am wondering, instead of mixing the logic between `test.yml` and `Dangerfile`, whether it's a good idea to keep it in a single place -  either in `test.yml` or in `Dangerfile`. As `test.yml` already has an `if` condition, we can add our new conditionals there. The updated `if` condition might look like:

```yml
if: github.event_name == 'pull_request' && contains( github.event.pull_request.labels.*.name, 'wip')  && contains( github.event.pull_request.title.*.name, 'wip') && github.event.pull_request.draft == false
```

@malparty ,  @sanG-github ,  would love to hear what you guys think of it as well, as you already commented on this PR!

Reply:
Moving the condition into the Github action has the advantage to not add any "GitHub" specific code in the DangerFile (which reduce cleaning required when a team is using GitLab or BitBucket) 🤔 

So I agree with this suggestion :)

🙈  The condition should be reversed though:
```
if: github.event_name == 'pull_request' && !contains( github.event.pull_request.labels.*.name, 'wip')  && !contains( github.event.pull_request.title.*.name, 'wip') && github.event.pull_request.draft == false
```

Reply:
Thanks for the suggestion 🙏🏼 . Fixed in d57cfc1d3a4a77beb4b093983ad64fa41ddcc0e3

Reply:
What a nice suggestion!! Thank Rajib 😘